### PR TITLE
Feature/global variables

### PR DESCRIPTION
**Pull request by iruzevic**
Create custom project color pallete. This colors are fetched from the main manifest.json file located in src>blocks folder.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 ### Changed
 * Removed config dependency from the Asset classes and exposed config through Manifest. 
-* `editor-color-pallete` - Add theme support for editor color pallete.
+* `editor-color-pallete` - Add theme support for editor color pallete built from global manifest.json.
```
Comment (volfkarlo) [inline CHANGELOG.md:10]: Palette*

File: src/blocks/class-blocks.php [hunk 2]
```diff
@@ -94,29 +94,18 @@ public function register() {
...
 
   /**
-   * Changes the default Gutenberg color pallete. Add your colors below.
-   * 
-   * !IMPORTANT - Regarding Eightshift custom blocks:
-   * - Color (when selected) should be added as element's modifier class like so:
-   *   block-class--white
-   * - You need an perform an additional step of styling (coloring) your attribute
-   *   based on color's slug class modifier in the block's `block-style.scss`
+   * Create custom project color pallete.
```
Comment (volfkarlo) [inline src/blocks/class-blocks.php:97]: palette*

File: src/blocks/class-blocks.php [hunk 3]
```diff
@@ -94,29 +94,18 @@ public function register() {
...
+   * This colors are fetched from the main manifest.json file located in src>blocks folder.
    *
    * @return void
    */
   public function change_editor_color_pallete() {
```
Comment (volfkarlo) [inline src/blocks/class-blocks.php:102]: palette*

File: src/blocks/class-blocks.php [hunk 4]
```diff
@@ -90,33 +90,22 @@ public function register() {
...
-        'color' => '#C3151B',
-      ),
-    ) );
+  public function change_editor_color_palette() {
+
+    $colors = $this->get_settings()['globalVariables']['colors'] ?? [];
+
+    if ( $colors ) {
+      \add_theme_support( 'editor-color-palette', $colors );
+    }
```
Comment (IvanGrginovInf) [inline src/blocks/class-blocks.php:108]: Wish I didn't refactor this like 2 weeks before 🙈 my bad 😄

File: CHANGELOG.md (hunk 1 above)
Comment (iruzevic) [reply CHANGELOG.md]: done

File: src/blocks/class-blocks.php (hunk 2 above)
Comment (iruzevic) [reply src/blocks/class-blocks.php]: done

File: src/blocks/class-blocks.php (hunk 3 above)
Comment (iruzevic) [reply src/blocks/class-blocks.php]: done

File: src/blocks/class-blocks.php (hunk 4 above)
Comment (iruzevic) [reply src/blocks/class-blocks.php:108]: No problem, this gave me a great idea how make this new cool feature.

**Pull request merged**